src/command.c

```diff
@@ -177,7 +177,7 @@ bool cmd_version(target_s *t, int argc, const char **argv)
 #else
```

Comment:
The copyright and license notices should probably get copied up into the BMDA block (it needs a major rework anyway, but that's another problem)